Packs/MailListener_-_POP3/Integrations/MailListener_POP3/MailListener_POP3.yml

```diff
@@ -1,6 +1,6 @@
 category: Messaging
 commonfields:
-  id: MailListener - POP3 Beta
+  id: MailListener - POP3
```

Comment:
not sure we can modify integration id and names. if not - change only display and we'll live with that